src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,26 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int DEFAULT_SIZE_OF_PASSWORD_AND_LOGIN = 6;
```

Comment:
better separate into 2 constants, in case at some point in the future they no longer will be the same, it will be easier to update values